### PR TITLE
Added Openstack backend

### PR DESCRIPTION
**Pull request by axelaris**
Hi there,
I've added some templates to generate also openstack manifests.

### CODE REVIEW

**Comment by axelaris** [comment]
Hi guys,
anybody home? ;-)

File: templates/sample_stubs/sample_openstack_stub.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    gateway: 10.0.16.1
+    reserved:
+    - 10.0.16.2 - 10.0.16.254
+    static:
+    - 10.0.17.1 - 10.0.17.100
+    dns:
+    - 10.0.0.2
+    cloud_properties:
+      subnet: YOUR_SUBNET_ID
+      security_group: YOUR_SECURITY_GROUP_NAME
```
Comment (KostyaSha) [inline templates/sample_stubs/sample_openstack_stub.yml]: `net_id:` is also required for `manual` network

**Comment by axelaris** [comment]
Hi @kirederik,
why do you closed this PR?

**Comment by kirederik** [comment]
Hi @axelaris 
This was an unexpected side effect. 
We got rid of the `develop` branch this afternoon and your PR was closed when we deleted such branch.
Could you please resubmit this pull request to `master`?

**Pull request closed without merging**